ingestion/src/metadata/ingestion/source/dashboard/mode/client.py

```diff
@@ -74,23 +74,18 @@ def fetch_all_reports(self, workspace_name: str) -> Optional[list]:
         Returns:
             dict
         """
-        try:
-            all_reports = []
-            response_collections = self.client.get(f"/{workspace_name}/{COLLECTIONS}")
-            collections = response_collections[EMBEDDED]["spaces"]
-            for collection in collections:
-                response_reports = self.get_all_reports_for_collection(
-                    workspace_name=workspace_name,
-                    collection_token=collection.get(TOKEN),
-                )
+        all_reports = []
```

Comment:
removing the try/except here since this is the initial step and we don't want to move forward in the ingestion if this step fails